prusti-tests/tests/parse/fail/time_resources.rs

```diff
@@ -0,0 +1,11 @@
+use prusti_contracts::*;
+
+// ignore-test: prusti-constract functions used outside specification check not yet implemented
```

Comment:
This is a good test to have in the future (also with other Prusti syntax functions), but I don't think we can perform it after parsing only. This needs at least the type check phase.